migrator/migrations/sqls/20200220183445-seed-up.sql

```diff
@@ -26,6 +26,7 @@ CREATE TABLE IF NOT EXISTS Hackathons (
     end_date timestamp NOT NULL,
     location varchar(255) NOT NULL,
     max_reg smallint,
+    regDeadline timestamp NOT NULL,
```

Comment:
Lol... I just realized you have two different naming conventions here (`regDeadline` vs `reg_deadline`)